src/components/nftForm/NFTForm.vue

```diff
@@ -117,16 +117,28 @@ export default {
       if (approvedError) {
         this.$modal.value.showModal({
           type: 'error',
-          options: { modalProps: { message: approvedError.message } },
+          options: {
+            modalProps: { message: approvedError.message, title: 'Error checking approved status' },
+          },
         })
         this.isLoading = false
         return
       }
-      const [uriError, response] = await asyncTryCatch(fetch, uri)
+      const [uriError, response] = await asyncTryCatch(
+        fetch,
+        `https://cors-anywhere.herokuapp.com/${uri}`,
```

Comment:
isn't this only needed for local testing?

Reply:
This is necessary for every request to recover the metadata information; I moved the base URL to an environment variable.